Assets/Scripts/Player/PlayerRaycasts.cs

```diff
@@ -0,0 +1,54 @@
+﻿using UnityEngine;
+
+public class PlayerRaycasts
```

Comment:
You can mark classes as static if they are only supposed to contain static members ie. `public static class PlayerRaycasts`